### DISCUSSION TITLE
`/playlists/me` respones is missing document `links` required for pagination

### DISCUSSION THREAD

**Discussion by lazulit3** [tidal-music/discussions#120] [Report bug]
The response data for the "Get current user's playlists" endpoint at `/playlists/me` does not include the `links` field in the response document.

Because the `next` link is not provided in the document `links`, the page cursor is unknown, and it is not possible to request additional pages to get the full collection of the user's playlists.

This endpoint is documented at: https://developer.tidal.com/apiref?spec=user-playlist-v2&ref=get-my-playlists-v2&at=THIRD_PARTY

### Context

- My Tidal account has more than 20 playlists
- When authenticated as my user and requesting this endpoint, 20 playlists are included in the response data
- This suggests that I am receiving a single page, but lack the cursor needed to request additional pages
- I have the `playlists.read` scope enabled in my authorization and in my OAuth client (developer app)

### Reproduction

I can reproduce this consistently.

If you use the `Authorize` and `Try it out` buttons @ https://developer.tidal.com/apiref?spec=user-playlist-v2&ref=get-my-playlists-v2&at=THIRD_PARTY, you should be able to reproduce the issue.

You can download the JSON response to `user_playlists.json` and try `cat user_playlists.json | jq .links` to verify the field is not there. (`jq` will report `null`)

(Note: I'm testing with `Access Tier: In Development` in case that means anything.)

**Comment by semohr** [comment]
Sound identical to #97 and might also be related to #119.

**Comment by duttonst** [comment]
I can reproduce this too. I thought this was fixed in [#97](https://github.com/orgs/tidal-music/discussions/97) - is this a regression?

**Comment by stefanmoro** [comment]
Should be fixed a while back. Closing.
